### ISSUE TITLE
Separately categorize same item with different tertiary stats

### ISSUE THREAD

**Issue by akbyrd** [arkayenro/arkinventory#1560]
**What version number you are using?**
3.09.63

**What game client are you playing? Retail, Burning Crusade, Classic / Live, PTR, Beta**
Retail

**What language is the game client set to?**
English

**Ask your question**
I have 2 of the same item with different tertiary stats. Is it possible to categorize these separately? I would like to add the item with the extra Speed stat to one category and the one with Leech to different category. Currently, assigning a custom category to one assigns the other as well.

**Comment by arkayenro** [comment]
i dont believe i ever got around to making category assignments that specific.  i think its still base item based.

rules are based on the full hyperlink though, specifically the outfit one.

your best bet at the moment would be to use one of the supported outfit mods and throw the items into an outfit so that a rule can pick it up and put it where you want it to go.

**Comment by akbyrd** [comment]
Unfortunately, only one of the items is in a set. The other is in a plain category for my "extra" gear.

Incidentally, when I right click an item and remove the category I assigned it to it removes the category from the bar the item is in instead of the item itself.

**Comment by arkayenro** [comment]
resetting the item back to its default category is done by clicking on the default category which is listed under the category its currently assigned to.

the description text on the default category should explain this, although if its picked up by a rule then you cant reset it and it doesnt explain that.

using the category menus off the item will do things with the category, not the item.  double check the description text on the remove action and youll see it states that it will remove the category from the bar, not the item from the category.
